grep/hooks/run

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
Discussed with @smacfarlane that hooks should not be shipped with packages that aren't for services.

The suggestion at https://github.com/habitat-sh/core-plans/pull/2328/files#r261642428 would avoid the need for a docker container, and would therefore avoid the need for this hook.